.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Feature Request
+description: Request new or improved functionality or changes to existing cuDF functionality
+title: "[FEA] "
+labels: ["feature request"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and fill out this feature request!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open feature requests](https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22%2Cimprovement%2Cenhancement) and have found no duplicates for this request
+
+  - type: dropdown
+    id: new_or_improvement
+    attributes:
+      label: Is this a new feature, an improvement, or a change to existing functionality?
+      options:
+        - New Feature
+        - Improvement
+        - Change
+    validations:
+      required: true
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this feature request?
+      options:
+        - Critical (currently preventing usage)
+        - Medium
+        - Low (would be nice)
```

Comment:
Can we just use descriptive terms here? That way they are less loaded, but still allow the reporter to indicate priority.

Reply:
What are you thinking of for the descriptions? I like the clear `Critical` > `Medium` > `Low` and going with only descriptive could make people spend more cycles on thinking than a really straightforward 1/2/3 option.

(I also like staying consistent with cuSpatial/Morpheus, but can be convinced.)

Reply:
Tbh I like how there's nothing for medium - it makes it so we don't have to define, and it gives a clear "I really want this, but it doesn't stop me from using cuDF" indication.